orttraining/orttraining/training_ops/cuda/optimizer/lamb_impl.cu

```diff
@@ -609,7 +617,12 @@ CudaKernel::CudaAsyncBuffer<LambMultiTensorSyncRangeAndLock> compute_tensor_rang
 }
 
 template <typename TIn1, typename TIn2, typename TOut1, typename TOut2, typename TBuf>
-void LambMultiTensorReductionFunctor<TIn1, TIn2, TOut1, TOut2, TBuf>::operator()(cudaStream_t stream, ChunkGroup<4> chunk_group, const CudaKernel& kernel, void* reduction_buffer, size_t reduction_buffer_size) {
+void LambMultiTensorReductionFunctor<TIn1, TIn2, TOut1, TOut2, TBuf>::operator()(
+    cudaStream_t stream,
+    ChunkGroup<4> chunk_group,
```

Comment:
no necessary for this PR, but we should replace the 4 with an appropriate macro like for adamw

Reply:
There is a todo comments I added. Also there is vs item related to it.